### PR TITLE
chore:  Create manualTestingGuide.md

### PR DESCRIPTION
**Pull request by MindEvo**
Updated version of the manual testing guide. Moved some sections around and cleaned up some of the punctuation and wording.

### CODE REVIEW

File: docs/testing.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+For non-developing team members to conduct successful manual testing of the platform.
+
+## Mission
+* Work with developers
+* Test new features
+* Test release candidate
+
+## How to Get the Job Done
+### 01) Overview
+Comment observations in GitHub issues that you are working on. If the issue is new/needs special attention, create a new issue for it. If that issue happens that it had been reported earlier, you will be notified by the developers.
```
Comment (Dschoordsch) [inline docs/testing.md:11]: -1 Some of these sentences appear to not make sense
> If that issue happens that it had been reported earlier, you will be notified by the developers.

could you go through the text and make sure you can understand everything? If something doesn't make sense, feel free to just remove it.

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

File: docs/testing.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+## Manual Testing Guide
```
[the remark this answers is not in the document]
Comment (MindEvo) [reply docs/testing.md]: Fixed

File: docs/manualTestingGuide.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+* yarn dev
+
+To test the new feature or update, load your web browser and goto localhost:3000.
+
+**_TIP: Trace the respective branch where the changes on the issue are pushed by the developer. Pull it, test and comment if expected behavior is there._**
+
+### 03) Release Testing
+Release tests are completed through the staging environment and are done weekly (Tues/Weds depending on your timezone). The purpose of these is to identify issues that can block the release. Use the [release test guide](https://github.com/ParabolInc/parabol/issues/6155#issue-1157867836) as a template for testing. Copy the release test checklist directly into the release test thread as a comment and check each box as they are completed. Comment on any unexpected or unwanted behaviors. Running live demos of the current release from production will help you to identify any unexpected behavior(s) that may show up in the release tests.
+
+* Release Blocking features/issues: Unexpected behaviors in the current release that are not in the production environment.
```
Comment (Dschoordsch) [inline docs/manualTestingGuide.md:45]: +1 Blocking/non-blocking is decided by the release owner. For each release one developer is responsible (the release-owner). They will create the release test issue and should notify the testers when staging environment is ready for testing.

**Review by Dschoordsch — APPROVED** [review]

File: docs/manualTestingGuide.md (hunk 3 above)
Comment (MindEvo) [reply docs/manualTestingGuide.md]: Fixed

**Pull request merged**